lib/prelude/text/system.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:backbone/filter.dart';
+import 'package:backbone/prelude/text/trait.dart';
+import 'package:backbone/prelude/transform.dart';
+import 'package:backbone/realm.dart';
+import 'package:flame/components.dart';
+
+/// Handle any text rendering including, zoom, position and padding
+/// The node requires to have the TextTrait and TransformTrait and one TextComponent
+/// If multiple are found only the first is used
+void textSystem(Realm realm) {
+  final query = realm.query(Has([TextTrait, TransformTrait]));
+  final cameraZoom = realm.gameRef.camera.zoom;
+  final counterZoom = Vector2.all(cameraZoom + (-2.0 * (cameraZoom - 1.0)));
+
+  for (var node in query) {
+    final trait = node.get<TextTrait>();
+    if (trait.dirty) {
+      final textChildList = node.findChildren<TextComponent>();
+      if (textChildList.isNotEmpty) {
+        final textChild = textChildList.first;
+        textChild.text = trait.text;
+        textChild.textRenderer = TextPaint(style: trait.style);
+
+        final textSize = textChild.textRenderer.measureText(textChild.text);
+        if (trait.counterZoom) {
```

Comment:
What's `counterZoom`?

Reply:
Added comment, but as the name suggest if true the text will counter any active zoom to be always at the same size.